tests/repositories/user.repository.2.test.js

```diff
@@ -0,0 +1,40 @@
+const mockFindOne = jest.fn()
+jest.mock('../../models/user.model', () => {
+    return {
+        findOne: mockFindOne
```

Comment:
A placeholder for the function that you want to mock/stub.

Notice how we use this `mockFindOne` in 2 different ways in the 2 tests below